src/main/resources/templates/client/hint/list.html

```diff
@@ -1,10 +1,91 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="cz" xmlns:th="http://www.w3.org/1999/xhtml">
 <head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <link rel="stylesheet" th:href="@{/css/style.css}"/>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta charset="UTF-8">
-    <title>Nápovědy</title>
+
+    <title>Nápověda</title>
 </head>
 <body>
+    <nav class="navbar navbar-default navbarRed" >
+        <div class="navbarContentCentered">
+            <p class="bold floatLeft" th:text="${team.name}"></p>
+            <p class="floatLeft">Body:xx b</p>
+        </div>
+        <button id="navbarButton" class="btn" type="button" data-toggle="collapse" data-target="#navbarSupportedContent1"
+                aria-controls="navbarSupportedContent1" aria-expanded="false" aria-label="Toggle navigation"><span class="dark-blue-text">
+                    <i class="fa fa-bars"></i></span>
+        </button>
+        <div class="collapse navbar-collapse collapsedNavbar" id="navbarSupportedContent1">
+            <ul class="navbar-nav mr-auto">
+                <li class="nav-item collapsedNotActive">
+                    <a class=" nav-link" href="#">Šifry </a>
+                </li>
+                <div class="dropdown-divider-cst"></div>
```

Comment:
Tag div není správně v seznamu ul validní. Nejlepší by bylo to odstranit a ty separátory vyřešitpomocí CSS.

Reply:
Martin říkal, že si to CSS v klientské části upraví podle sebe, má na to issue https://github.com/JitkaCNGroup/lentils/issues/126. Mám to teda nějak měnit v tomto pull req, nebo mu to ponechat?

Reply:
Jo, potom jsme se bavili, že to udělá naráz. Takže pohoda, neřeš.